### PR TITLE
Rice2000 prot13 auth

### PR DESCRIPTION
**Pull request by rice2000**

### CODE REVIEW

File: content/docs/glossary/accounts.mdx [hunk 1]
```diff
@@ -53,8 +53,8 @@ To add a home domain to an account, use the [Set Options](link) operation.
 
 Asset issuers set flags at the account level if they want to control access to the assets they issue. There are three flags:
 
-- **Authorization required (0x1)**: Requires the issuing account to grant an account permission to hold an asset.
-- **Authorization revocable (0x2)**: Allows the issuing account to revoke an asset held by other accounts.
+- **Authorization required (0x1)**: Requires the issuing account to grant an account permission to hold an asset.  With this flag set, an issuer can either grant full authorization to transact with its asset *or* it can grant limited authorization allowing the holder to maintain orders on the order books, but not to otherwise transact with the asset.  
+- **Authorization revocable (0x2)**: Allows the issuing account to revoke an asset held by other accounts, or to reduce authorization from full to partial.
```
Comment (jonjove) [inline content/docs/glossary/accounts.mdx:57]: It seems like this was wrong before. Authorization revocable doesn't allow the issuer to revoke an asset held by other accounts (see https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/stellar-dev/hPhkXhrl5-Y/tt8cnMY5AgAJ for a discussion on this feature). Instead, authorization revocable allows the issuer to reduce the authorization level of an account from full to partial, from full to none, or from partial to none.

File: content/docs/start/list-of-operations.mdx [hunk 2]
```diff
@@ -381,11 +381,15 @@ Possible errors:
 ## Allow Trust
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.allowTrust) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/AllowTrustOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#AllowTrust)
 
-Updates the `authorized` flag of an existing trustline. This can only be called by the issuer of a trustline's [asset](./assets.md), and only when `AUTHORIZATION REQUIRED` (at the minimum) has been set on the issuer's account.
+Updates the `authorized` flag of an existing trustline. 
 
-The issuer can only clear the `authorized` flag if the issuer has the `AUTH_REVOCABLE_FLAG` set. Otherwise, the issuer can only set the `authorized` flag.
+This can only be called by the issuer of a trustline's [asset](./assets.md), and only when `AUTHORIZATION REQUIRED` (at the minimum) has been set on the issuer's account.
```
Comment (jonjove) [inline content/docs/start/list-of-operations.mdx:386]: This is unrelated to the changes in this PR, but why does it add "(at the minimum)" here? What's the meaning of that?

File: content/docs/glossary/accounts.mdx (hunk 1 above)
Comment (rice2000) [reply content/docs/glossary/accounts.mdx]: Great!  I pushed that change.

File: content/docs/start/list-of-operations.mdx (hunk 2 above)
Comment (rice2000) [reply content/docs/start/list-of-operations.mdx]: Also a good point.  I deleted that + pushed the change.

**Pull request merged**